### PR TITLE
Update architecture.md

### PR DESCRIPTION
**Pull request by ronitbiswas97**
Signed-off-by: Ronit Biswas <ronitbiswas97@gmail.com>

### CODE REVIEW

File: docs/architecture.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ During installation, following three CRDs are installed on the Kubernetes cluste
 
 **Chaos-Experiments**
 
-Chaos Experiment is a CR and are available as YAML files on <a href=" https://hub.litmuschaos.io" target="_blank">Chaos Hub</a>. For more details visit Chaos Hub [documentation](chaoshub.html).
+Chaos Experiment is a CR and are available as YAML files on <a href=" https://hub.litmuschaos.io" target="_blank">Chaos Hub</a>. For more details visit Chaos Hub [documentation](../chaoshub).
```
Comment (chandankumar4) [inline docs/architecture.md:28]: Thanks for your contribution @ronitbiswas97, Can you verify the link once, I think the file location is not correct

**Comment by anupriya0703** [comment]
LGTM

**Review by anupriya0703 — APPROVED** [review]
LGTM

**Comment by chandankumar4** [comment]
Thanks @ronitbiswas97 for your contribution but these changes are not required

**Pull request closed without merging**